corehq/apps/data_cleaning/templates/data_cleaning/columns/column_editable.html

```diff
@@ -2,8 +2,73 @@
 {% load data_cleaning %}
 
 {% edited_value record column as edited_value %}
+<div
+  class="editable-column-container"
+  x-data="{
+    isEditing: false,
+    isSubmitting: false,
+  }"
+>
+  <div
+    class="dc-inline-edit-block"
+    x-show="!isEditing"
+    @dblclick="isEditing = !isSubmitting"
+  >
+    {% if edited_value|has_edits %}
+      <div class="d-flex align-items-center">
+        <div class="dc-diff-view">
+          <div class="dc-diff-before">
+            {{ value }}
+          </div>
+          <div class="dc-diff-after">
+            {{ edited_value }}
+          </div>
+        </div>
+        <div class="ms-2">
+          <button
+            class="btn btn-outline-danger btn-sm htmx-loading"
```

Comment:
Do we still need `htmx-loading` here?

Reply:
https://github.com/dimagi/commcare-hq/pull/36247/commits/2aad6b1bffe9790152d5dd584e305e24b3ed1dbf